src/components/buttonMenu/buttonMenu.svelte

```diff
@@ -40,7 +41,7 @@
     <slot name="anchor-content">Click</slot>
   </div>
   <Menu
-    bind:isOpen
+    bind:isOpenInternal
```

Comment:
I don't think you can bind `isOpenInternal` like this - what's it binding to?

Reply:
Additionally, maybe we shouldn't use bind here at all? What if the API on Menu changes and it doesn't emit an onClose - `isOpenInternal` would change but our `isOpen` state wouldn't get updated (and we might not emit our close event).

Reply:
fixed